### PR TITLE
[BugFix]Create customer for connected account instead of platform

### PR DESCRIPTION
**Pull request by apoorv-mishra**
## Notion card
NA
## Summary

Fix to create customer for connected account instead of the platform. Refer [docs](https://stripe.com/docs/connect/authentication) for the right ways to make API calls for connected accounts.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [x] I have manually tested all workflows
- [x] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

File: app/models/client.rb [hunk 1]
```diff
@@ -121,6 +121,8 @@ def register_on_stripe!
           metadata: {
             platform_id: id
           }
+        }, {
+          stripe_account: stripe_connected_account.account_id
```
Comment (akhilgkrishnan) [inline app/models/client.rb:125]: can you add a test for this?

**Review by apoorv1316 — APPROVED** [review]
Tested and payment was successful in local

**Review by shivamsinghchahar — APPROVED** [review]
@apoorv-mishra Nice catch! LGTM

File: app/models/client.rb (hunk 1 above)
Comment (apoorv-mishra) [reply app/models/client.rb:125]: Added.

**Pull request merged**